src/main/java/redis/clients/jedis/commands/MultiKeyJedisClusterCommands.java

```diff
@@ -32,15 +32,19 @@
 
   String lmove(String srcKey, String dstKey, ListDirection from, ListDirection to);
 
-  String blmove(String srcKey, String dstKey, ListDirection from, ListDirection to, int timeout);
+  String blmove(String srcKey, String dstKey, ListDirection from, ListDirection to, double timeout);
 
   List<String> blpop(int timeout, String... keys);
 
+  KeyedListElement blpop(double timeout, String... keys);
+
   List<String> brpop(int timeout, String... keys);
 
-  KeyedTuple bzpopmax(int timeout, String... keys);
+  KeyedListElement brpop(double timeout, String... keys);
```

Comment:
@sazzad16 I think you should deprecate the interface methods and not only the implementation methods

Reply:
@gkorland The deprecated methods were not part of any interface.